### DISCUSSION TITLE
Ram usage in pulsar

### DISCUSSION THREAD

**Discussion by ghost** [pulsar-edit/.github#83] [General]
How much ram used in. Pulsar Hackable editor...

**Comment by confused-Techie** [comment]
That's hard to say exactly how much RAM will be used as that will depend on your setup and the packages you have enabled.

But as of now, it should be roughly equivalent to the RAM usage you'd see while using Atom.

Unless anyone else has any notes on this, I'd say the best bet is to do some testing with our most recent binaries to find out.
